src/main/java/org/trips/service_framework/utils/CmsUtils.java

```diff
@@ -0,0 +1,47 @@
+package org.trips.service_framework.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.util.CollectionUtils;
+import org.trips.service_framework.constants.CmsConstants;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+
+public class CmsUtils {
+
+    public static String sanitizeInput(String attribute, String value) {
+        if (Objects.isNull(value) || value.isEmpty() || value.equals(CmsConstants.NOT_APPLICABLE)) {
+            return CmsConstants.NOT_APPLICABLE;
+        }
+
+        if (attribute.equals("treatment")) {
+            return sanitizeTreatment(value);
+        }
+
+        return value.trim();
+    }
+
+    private static String sanitizeTreatment(String treatment) {
+        List<String> treatmentArray = new ArrayList<>();
```

Comment:
comment one example here of what this method is doing.

Reply:
In our database, we found several formats for treatment such as:
- null
- abc
- abc, def
- [abc, def]
- [“abc”, “def”]

To standardize these treatment attribute name, we will sanitise the treatment attribute in this`["abc", "def"]` format.